apps/src/StudioApp.js

```diff
@@ -643,8 +643,15 @@ StudioApp.prototype.initProjectTemplateWorkspaceIconCallout = function() {
   }
 };
 
+// When pairing, source code is stored only with the driver. If the user completed
+// this level as a navigator, show an alert with a link to the (read-only) source
+// code stored in the driver's account.
 StudioApp.prototype.alertIfCompletedWhilePairing = function(config) {
-  if (!!config.level.pairingDriver) {
+  if (!config.level.isNavigator) {
+    return;
+  }
+
+  if (config.level.pairingDriver) {
```

Comment:
Is `pairingDriver` populated if the driver is logged in? If so, this alert would have displayed for the driver before as well, which it wouldn't now. Just want to make sure this is the intended behavior.

Reply:
No, the previous implementation would return nil for pairingDriver.  It went through UserLevel#driver_user_levels which would be nil for the driver's user_level.

One thing to note is, previously, existence of pairingDriver indicated whether the the current user was a navigator.  I had to introduce isNavigator to distinguish the case where the current user is a navigator but we don't have the driver's name.